.packit.yaml

```diff
@@ -5,6 +5,9 @@ synced_files:
 # https://packit.dev/docs/configuration/#top-level-keys
 downstream_package_name: python-ogr
 upstream_project_url: https://github.com/packit-service/ogr
+# we are setting this so we can use packit from ogr's dist-git
+# packit can't know what's the upstream name when running from distgit
```

Comment:
What trying repo name?

Reply:
it was failing on the fact that tarball name started as python-ogr, instead of ogr - when I ran commands from dist-git

Reply:
That was just an idea of what "magic" we can do in packit. (But by default, we pick the downstream name so it will collide with this approach.)